image_test.go

```diff
@@ -44,7 +45,7 @@ func TestMain(m *testing.M) {
 		return regularTermination
 	}
 	if err := Run(f, 320, 240, 1, "Test"); err != nil && err != regularTermination {
-		panic(err)
+		log.Fatal(err)
```

Comment:
Why is log preferred?

Reply:
that's something that I prefer, you also get the os.exit(1) + error without all the stack trace from panic